[docs][Menu][joy] Explain how to control the open state (#38355)
mui/material-ui: docs/data/joy/components/menu/menu.md

{{"demo": "GroupMenu.js"}}

### Controlling the open state

By default, the open/close state of the menu is managed internally by the Dropdown component.

> It is possible to control it programmatically from the outside, though.
> To do this, use the `open` and `onOpenChange` props of the Dropdown component, as shown in the demo:

Reviewer: Copyediting nitpick to condense the section

```suggestion
To control it programmatically from the outside, apply the Dropdown's `open` and `onOpenChange` props as shown below:
```
